app/src/main/cpp/src/components/utils/CMakeLists.txt

```diff
@@ -31,17 +31,21 @@
 include(${CMAKE_SOURCE_DIR}/tools/cmake/helpers/platform.cmake)
 include(${CMAKE_SOURCE_DIR}/tools/cmake/helpers/sources.cmake)
 
+find_package(Sqlite3 REQUIRED)
```

Comment:
@ychernysheva should this be moved to lines 46-48 and added as `else()` branch to new if statement? I believe this also should be wrapped into ANDROID ifdef

Reply:
@AKalinich-Luxoft Done in 6ccfc841a7429d1da9b7d590615a29ba744ccd1a